html-forms/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form>
+    <div>
+      <label for="email">Email:
+        <br>
+        <input type="email">
+      </label>
+    </div>
+    <div>
+      <label for="dropdown">Why is coding cool?
+        <br>
+        <select name="reasons" id="selector_reasons" size="1">
+          <option value="select">Select an option</option>
+          <option value="interesting">Because it's interesting!</option>
+          <option value="fun">Beacuse it's fun!</option>
+          <option value="powerful">Because it's powerful!</option>
+        </select>
+      </label>
+    </div>
+    <div>
+      <label for="radiochoices">Favorite Language (choose one)
```

Comment:
instead of one label to wrap all of the radio inputs, I would wrap a label around each one individually